README.md

```diff
@@ -34,115 +63,157 @@ You can also tab complete remote names and branch names e.g.: `git pull or<tab>
     • [CHANGELOG](https://github.com/dahlbyk/posh-git/blob/v0.7.0/CHANGELOG.md) )
 
 ## Notes
+
 posh-git adds variables to your session to let you customize it, including `$GitPromptSettings`, `$GitTabSettings`, and `$TortoiseGitSettings`.
 For an example of how to configure your PowerShell profile script to import the posh-git module and create a custom prompt function that displays git status info, see the [Customizing Your PowerShell Prompt](https://github.com/dahlbyk/posh-git#step-3-optional-customize-your-powershell-prompt) section below.
 
 Note on performance: Displaying file status in the git prompt for a very large repo can be prohibitively slow.
 Rather than turn off file status entirely (`$GitPromptSettings.EnableFileStatus = $false`), you can disable it on a repo-by-repo basis by adding individual repository paths to `$GitPromptSettings.RepositoriesInWhichToDisableFileStatus`.
 
 ## Installation
+
 ### Prerequisites
+
 Before installing posh-git make sure the following prerequisites have been met.
 
-1. PowerShell 3.0 or higher. Check your PowerShell version by executing `$PSVersionTable.PSVersion`.
+1. Windows PowerShell 5.x or PowerShell Core 6.0.
+   Get PowerShell Core 6.0 for Windows, Linux or macOS from [here][pscore-install].
+   You can check your PowerShell version by executing `$PSVersionTable.PSVersion`.
 
-2. Script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
+1. On Windows, script execution policy must be set to either `RemoteSigned` or `Unrestricted`.
    Check the script execution policy setting by executing `Get-ExecutionPolicy`.
    If the policy is not set to one of the two required values, run PowerShell as Administrator and execute `Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Confirm`.
 
-3. Git must be installed and available via the PATH environment variable.
+1. Git must be installed and available via the PATH environment variable.
    Check that `git` is accessible from PowerShell by executing `git --version` from PowerShell.
    If `git` is not recognized as the name of a command verify that you have Git installed.
    If not, install Git from [https://git-scm.com](https://git-scm.com).
    If you have Git installed, make sure the path to git.exe is in your PATH environment variable.
 
-### Installing posh-git via PowerShellGet
-If you are on PowerShell version 5 or higher, execute the command below to install from the [PowerShell Gallery](https://www.powershellgallery.com/):
+### Installing posh-git
+
+posh-git is available on the [PowerShell Gallery][psgallery] and can be installed using the PowerShellGet module.
+Start either Windows PowerShell 5.x or PowerShell Core 6.x (`pwsh`) and execute the following command:
 
 ```powershell
-PowerShellGet\Install-Module posh-git -Scope CurrentUser
+Install-Module posh-git -Scope CurrentUser -Force
```

Comment:
`-Force` to get latest version?

Reply:
To force side-by-side install. If you already have 0.7.1 installed, Install-Module will complain IIRC.  Update-Module posh-git would work if you installed from the PSGallery. Or just use Install-Module with -Force.
```
WARNING: Version '0.7.1' of module 'posh-git' is already installed at
'C:\Users\Keith\Documents\WindowsPowerShell\Modules\posh-git\0.7.1'. To install version '1.0.0-beta1', run
Install-Module and add the -Force parameter, this command will install version '1.0.0-beta1' in side-by-side with
version '0.7.1'.
```

Reply:
I'll defer to your experience here.